### PR TITLE
add indentation to main.html file

### PR DESCRIPTION
**Pull request by SeriyBg**
According to the discussion in PR #308 I have added indentation to the main.html file
Also noticed that there were not closed `<pre>` tag which was ignored in the browser so I have removed it as well

### CODE REVIEW

**Comment by ldemailly** [comment]
tags are closed in the code. please make sure you test your change and valide the resulting html

**Comment by SeriyBg** [comment]
@ldemailly oh, I see. Sorry, my bad. Have restored the removed tags, so only the whitespace changes are added

File: ui/templates/main.html [hunk 1]
```diff
@@ -1,94 +1,100 @@
...
+  <script src="{{.ChartJSPath}}"></script>
+  <script src="{{.Version}}/static/js/fortio_chart.js"></script>
+  <link rel="icon" href="../favicon.ico"/>
 </head>
 <body style="background: linear-gradient(to right, #d8aa20 , #c75228);">
-<a href="https://fortio.org/" target="_blank"><img src="{{.LogoPath}}" alt="." height="69" width="45" align="right" /></a>
-<img src="{{.LogoPath}}" alt="." height="92" width="60" align="right" />
-<img src="{{.LogoPath}}" alt="istio logo" height="123" width="80" align="right" />
+<a href="https://fortio.org/" target="_blank"><img src="{{.LogoPath}}" alt="." height="69" width="45"
+                                                   align="right"/></a>
```
Comment (ldemailly) [inline ui/templates/main.html]: put that one back on 1 line please, it's not more readable with align=right... on next line (it's ok to have more than 80 cols line for html)

File: ui/templates/main.html [hunk 2]
```diff
@@ -1,94 +1,100 @@
...
-Debug fetch http://<input type="text" id="uri" name="uri" value="" size=50/>
-</div>
+  <div>
+    Debug fetch http://<input type="text" id="uri" name="uri" value="" size=50/>
+  </div>
 </form>
 <p><i>Or</i></p>
-<a href="{{.DebugPath}}">debug</a> and <a href="{{.DebugPath}}?env=dump">debug with env dump</a> and <a href="../debug/pprof/">Internal PPROF</a>
+<a href="{{.DebugPath}}">debug</a> and <a href="{{.DebugPath}}?env=dump">debug with env dump</a> and <a
+    href="../debug/pprof/">Internal PPROF</a>
```
Comment (ldemailly) [inline ui/templates/main.html]: please move <a on this same line

File: ui/templates/main.html [hunk 3]
```diff
@@ -1,94 +1,100 @@
...
+                                                   align="right"/></a>
+<img src="{{.LogoPath}}" alt="." height="92" width="60" align="right"/>
+<img src="{{.LogoPath}}" alt="istio logo" height="123" width="80" align="right"/>
 <h1>Φορτίο (fortio) v{{.Version}}{{if not .DoLoad}} control UI{{end}}</h1>
 <p>Up for {{.UpTime}} (since {{.StartTime}}).
 {{if .DoLoad}}
 <p>{{.Labels}} {{.TargetURL}}
 <div id="running">
-Running load test... Results pending...
-<br />
```
Comment (ldemailly) [inline ui/templates/main.html:13]: please leave as xhtml `<br />` (or we could switch to plain html `<br>` but not `<br/>`)
https://www.quora.com/What-is-the-difference-between-br-and-br-tag-in-HTML-Which-one-should-we-use

also please run through weblint (from the running fortio server) before/after

**Review by ldemailly — CHANGES_REQUESTED** [review]

**Comment by ldemailly** [comment]
also avoid force pushing, please make multiple commits so the history during review is preserved, it gets squashed at the end anyway on master

File: ui/templates/main.html (hunk 1 above)
Comment (SeriyBg) [reply ui/templates/main.html]: Done

File: ui/templates/main.html (hunk 3 above)
Comment (SeriyBg) [reply ui/templates/main.html:13]: Done

**Comment by SeriyBg** [comment]
Ok, clear. I thought that it is better to keep the commits history clear, but will make a separate commit as log as to master it squashed as one merge commit

**Review by ldemailly — APPROVED** [review]

**Comment by ldemailly** [comment]
Sorry for the delay in getting this in - thanks!

**Comment by SeriyBg** [comment]
@ldemailly no worries, thanks for accepting it!

**Pull request merged**